packages/stark-ui/src/modules/toast-notification/components/_toast-notification.component.scss

```diff
@@ -0,0 +1,43 @@
+.mat-snack-bar-container {
+  margin: 8px !important;
+  padding: 0 !important;
+}
+
+.stark-toast {
+  min-height: 0;
+  display: flex;
+  white-space: normal;
+  align-items: center;
+  padding: 6px 16px 6px 16px;
+  & span {
+	line-height: 24px;
+	margin: 6px;
+  }
+  & mat-icon {
+	margin: 6px;
+	min-width: 24px;
+	width: 24px;
+	min-height: 24px;
+	height: 24px;
+  }
+  & .stark-toast-action {
```

Comment:
the & is not necessary in scss in this case.
It depends on the result needed in CSS: 
* `.stark-toast .stark-toast-action{ }` gives this in SCSS:
```scss
.stark-toast {
  .stark-toast-action { }
}
```
* `.stark-toast.stark-toast-action{ }` gives this in SCSS:
```scss
.stark-toast {
  &.stark-toast-action { }
}
```